src/app/main-nav/main-nav.component.html

```diff
@@ -1,29 +1,28 @@
 <mat-sidenav-container class="sidenav-container">
-  <mat-sidenav #drawer class="sidenav" fixedInViewport [attr.role]="(isHandset$ | async) ? 'dialog' : 'navigation'"
-    [mode]="(isHandset$ | async) ? 'over' : 'side'" [opened]="(isHandset$ | async) === false">
+  <mat-sidenav #drawer class="sidenav" mode="side" opened="true">
     <mat-toolbar>Menu</mat-toolbar>
     <mat-nav-list>
-      <a mat-list-item routerLink="home">Home</a>
+      <a mat-list-item routerLink="home">Builders</a>
       <a mat-list-item routerLink="reference">References</a>
     </mat-nav-list>
   </mat-sidenav>
   <mat-sidenav-content>
     <mat-toolbar color="primary">
-        <mat-toolbar-row class="container">
-          <button type="button" aria-label="Toggle sidenav" mat-icon-button (click)="drawer.toggle()"
-            *ngIf="isHandset$ | async">
-            <mat-icon aria-label="Side nav toggle icon">menu</mat-icon>
-          </button>
-          <span>Angular Builders</span>
-          <span class="example-spacer"></span>
-          <a href="https://github.com/santoshyadav198613/angular-builder" mat-button class="github-button">
-            <span>
-              <img class="github-icon"
-                src="https://material.angular.io/assets/img/homepage/github-circle-white-transparent.svg" alt="Github">
-              Github
-            </span>
-          </a>
-        </mat-toolbar-row>
+      <mat-toolbar-row>
+        <button type="button" aria-label="Toggle sidenav" mat-icon-button (click)="drawer.toggle()"
+          *ngIf="isHandset$ | async">
+          <mat-icon aria-label="Side nav toggle icon">menu</mat-icon>
+        </button>
+        <span>Angular Builders</span>
+        <span class="example-spacer"></span>
+        <a href="https://github.com/santoshyadav198613/angular-builder" mat-button class="github-button">
+          <span>
+            <img class="github-icon"
```

Comment:
The icon isn't centered vertically. Using `mat-icon` and its `getSvgIconFromUrl()` might help.